build.gradle

```diff
@@ -84,9 +85,38 @@ test {
 
 task functional(type: Test) {
   description = "Runs functional tests"
-  group = "Verification"
+  group = "verification"
+  dependsOn assemble, testClasses
   testClassesDirs = sourceSets.functionalTest.output.classesDirs
-  classpath = sourceSets.functionalTest.runtimeClasspath
+  classpath = configurations.functionalTestRuntimeClasspath + sourceSets.functionalTest.output
+
+  useJUnitPlatform {
+    includeTags System.getProperty("tags", "Functional")
+  }
+
+  doFirst {
```

Comment:
This needs potential cleanup, no need of all this to be run on Jenkins. A local script should do if needed for local. Below steps are tedious and should be manageable by `serenity.properties`.